tests/unit/core/reducers/test_addons.js

```diff
@@ -105,4 +112,18 @@ describe('addon reducer', () => {
       baz: { description: null },
     });
   });
+
+  it('exposes "isRestartRequired" attribute from current version files', () => {
```

Comment:
This needs a test to make sure it sets `isRestartRequired` when at least one of the files declares it. You could modify the helper a bit and set it up like:

```js
const addon = createFakeAddon({
  files: [
    { ...fakeAddon.currentVersion.files[0], is_restart_required: false },
    { ...fakeAddon.currentVersion.files[0], is_restart_required: true },
  ],
});
```